src/status_im/contexts/wallet/common/utils.cljs

```diff
@@ -494,10 +494,12 @@
     (sort-by (juxt :symbol priority) tokens)))
 
 (defn tokens-with-balance
-  [tokens networks chain-ids]
+  [tokens networks token-supported-chains chain-ids]
```

Comment:
When I read name `token-supported-chains` my brain splits this like "[token-supported] [chains]" which is quite confusing because I can't figure out what are chains that are supported by tokens :)
I'd suggest to rename all similar variables, keywords and functions (17 entries) to make them easier to understand:
'token-supported-chains' -> 'chains-supported-by-token'
'token-supported-chains-by-symbol' -> 'supported-chains-by-token-symbol'
Wdyt?